src/base/memory_monitor_sr.cpp

```diff
@@ -86,8 +86,9 @@ namespace cubperf
       mmon_component &operator = (mmon_component &&) = delete;
 
       const char *get_name ();
-      void add_stat (uint64_t size, int subcomp_idx, bool init, bool expand);
-      void sub_stat (uint64_t size, int subcomp_idx, bool init);
+      void add_stat (uint64_t size, int subcomp_idx, bool allocation_at_init);
```

Comment:
Nit. to me, `allocation` seems redundant.
```suggestion
      void add_stat (uint64_t size, int subcomp_idx, bool init);
```

Reply:
For clear understanding, I want to remain this parameter name.